src/test/orphanage_tests.cpp

```diff
@@ -126,13 +114,15 @@ BOOST_AUTO_TEST_CASE(DoS_mapOrphans)
         tx.vout[0].nValue = i*CENT;
         tx.vout[0].scriptPubKey = GetScriptForDestination(PKHash(key.GetPubKey()));
 
-        orphanage.AddTx(MakeTransactionRef(tx), i);
+        auto ptx = MakeTransactionRef(tx);
+        orphanage.AddTx(ptx, i);
+        orphans_added.emplace_back(ptx);
```

Comment:
nitty nit: for consistency, this should probably be done after every successful `AddTx` call (e.g. also in the next loop below), though it doesn't seem to be necessary for now

Reply:
added